Add missing javadoc

Leftover for https://github.com/checkstyle/checkstyle/issues/10818

```
[ERROR] src/main/java/com/github/sevntu/checkstyle/checks/coding/CustomDeclarationOrderCheck.java:[1323,9] (javadoc) MissingJavadocMethod: Missing a Javadoc comment.

[ERROR] src/main/java/com/github/sevntu/checkstyle/checks/coding/CustomDeclarationOrderCheck.java:[1327,9] (javadoc) MissingJavadocMethod: Missing a Javadoc comment.
```

--- a/sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/CustomDeclarationOrderCheck.java
+++ b/sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/CustomDeclarationOrderCheck.java
@@ -1320,10 +1320,20 @@ public class CustomDeclarationOrderCheck extends AbstractCheck {
          */
         private final List<DetailAST> setters = new LinkedList<>();
 
+        /**
+         * Returns the current position.
+         *
+         * @return the current position
+         */
         public int getCurrentPosition() {
             return currentPosition;
         }
 
+        /**
+         * Setter for the current position.
+         *
+         * @param position the new position
+         */
         public void setCurrentPosition(int position) {
             currentPosition = position;
         }

